specification/app/resource-manager/Microsoft.App/stable/2025-01-01/ManagedEnvironments.json

```diff
@@ -1359,6 +1359,11 @@
           "$ref": "./CommonDefinitions.json#/definitions/WorkloadProfileName",
           "description": "Workload profile type for the workloads to run on."
         },
+        "enableFips": {
+          "type": "boolean",
```

Comment:
you have one warning here : [https://github.com/Azure/azure-rest-api-specs/pull/31987/checks?check_run_id=35083982630](https://github.com/Azure/azure-rest-api-specs/pull/31987/checks?check_run_id=35083982630) 


Enums are preferred over boolean. For example its better and more extensible to do something like : 

FipsMode: Enabled\Disabled

so that later you can potentially add new states like Suspended etc

Reply:
@raosuhas This change is being promoted from our 2024-08-02-preview api-version. Refer PR https://github.com/Azure/azure-rest-api-specs/pull/30418
https://github.com/Azure/azure-rest-api-specs/blob/698431b03e6c6f2142c0edae33000902eb5340d2/specification/app/resource-manager/Microsoft.App/preview/2024-08-02-preview/ManagedEnvironments.json#L1831

We like to match the same property as aks. Refer https://github.com/Azure/azure-rest-api-specs/blob/698431b03e6c6f2142c0edae33000902eb5340d2/specification/containerservice/resource-manager/Microsoft.ContainerService/aks/stable/2024-09-01/managedClusters.json#L3459